doc/tutorials/workers.md

```diff
@@ -0,0 +1,114 @@
+# Workers
+
+**CDS** uses workers to execute pipelines. You will need, for sure, to create your own worker models.
```

Comment:
A pipeline is structured in sequential stages containing one or multiple concurrent jobs.
A Job will be executed by a worker.